### PR TITLE
NAT-3071 - Updates for iPhone X

### PR DESCRIPTION
**Pull request by jasonmerino**

### CODE REVIEW

File: src/ExNavigationBar.js [hunk 1]
```diff
@@ -8,11 +8,14 @@ import {
   TouchableOpacity,
   View,
   ViewPropTypes,
+  Dimensions,
```
Comment (gwmccull) [inline src/ExNavigationBar.js]: doesn't look like `Dimensions` and `PixelRatio` are used in this file

File: src/ExNavigationBar.js [hunk 2]
```diff
@@ -24,12 +27,19 @@ if (expoModule) {
...
+const IS_IPHONE_X = deviceUtils.isIphoneX();
+
 // Exponent draws under the status bar on Android, but vanilla React Native does not.
 // So we need to factor the status bar height in with Exponent but can ignore it with
 // vanilla React Native
-const STATUSBAR_HEIGHT = Platform.OS === 'ios'
-  ? 20
-  : global.__exponent ? 24 : 0;
+let STATUSBAR_HEIGHT = 0;
+if (Platform.OS === "ios" && IS_IPHONE_X) {
```
Comment (gwmccull) [inline src/ExNavigationBar.js]: single quotes on `"ios"` just for consistency

Comment (gwmccull) [reply src/ExNavigationBar.js]: line 36 & 38

**Review by gwmccull — CHANGES_REQUESTED** [review]

**Comment by jasonmerino** [comment]
@gwmccull Fixed up the code based on your comments.

File: src/tab/ExNavigationTab.js [hunk 3]
```diff
@@ -3,7 +3,7 @@
  */
 
 import React, { Children } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { StyleSheet, View, Platform, Dimensions, PixelRatio } from 'react-native';
```
Comment (gwmccull) [inline src/tab/ExNavigationTab.js]: Just noticed the unused variables here also.  Sorry for not catching the first time

File: src/tab/ExNavigationTabBar.js [hunk 4]
```diff
@@ -6,6 +6,9 @@ import {
   TouchableNativeFeedback,
   Text,
   View,
+  Platform,
```
Comment (gwmccull) [inline src/tab/ExNavigationTabBar.js]: Just noticed the unused variables here also.  Sorry for not catching the first time

**Review by gwmccull — CHANGES_REQUESTED** [review]

**Comment by jasonmerino** [comment]
@gwmccull good catches. It's amazing what I miss when refactoring without a linting tool...

**Review by gwmccull — APPROVED** [review]

**Pull request merged**